src/reducers/index.js

```diff
@@ -12,12 +12,17 @@ const blurType = blur().type;
 const focusType = focus().type;
 
 const applyFormReducer = (state, action={}) => {
-  if (action.type !== blurType && action.type !== focusType) {
+  switch(action.type) {
+  case focusType:
+  case blurType:
```

Comment:
The body of this case should really be wrapped in a block. `{ .... }`. `const` is a block scope variable. I know it doesn't make a huge difference here, but for correctness sake I think it shluld be wrapped. That is why I always just call a function in my switch statements.